.craft.yml

```diff
@@ -0,0 +1,11 @@
+minVersion: '0.7.0'
+github:
+  owner: getsentry
+  repo: sentry-php
+changelogPolicy: simple
+targets:
+  - name: github
+  - name: registry
+    type: sdk
+    config:
+      canonical: 'composer:sentry/sentry'
```

Comment:
Missing EOL here, the rest looks good to me apart from the indentation that I would change to 4 spaces to be consistent with everything else in this repository